creators/incofer/incofer.json

```diff
@@ -12,6 +12,13 @@
           "operator": "Incofer"
         }
     },
+    "routes": {
+        "color": "ff0000",
+        "description": "Esta línea está a prueba",
+        "text_color": "ffffff",
+        "type": "Tram",
+        "url": "http://www.incofer.go.cr/tren-urbano-alajuela-rio-segundo"
+    },
```

Comment:
This assumes that all lines have the same color, description, etc.
Is this really something for the config? Maybe these should be seen as default values, if no other source of information is available?

Reply:
I think @xamanu was advocating for storing these things in OSM, if I remember correctly.

Reply:
Indeed He was advocating. (I should add that data to osm)